client/me/pending-payments/pending-list-item.jsx

```diff
@@ -3,33 +3,69 @@
  * External dependencies
  */
 import React from 'react';
-import { localize } from 'i18n-calypso';
+import { localize, moment } from 'i18n-calypso';
 import Gridicon from 'gridicons';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import Card from 'components/card';
 import Button from 'components/button';
+import { getSite, getSiteTitle, getSiteUrl, getSiteDomain } from 'state/sites/selectors';
+import PurchaseSiteHeader from '../purchases/purchases-site/header';
+import { purchaseType } from 'lib/purchases';
+import { paymentMethodName } from 'lib/cart-values';
 
-export function PendingListItem( { productName, paymentType, totalCostDisplay, translate } ) {
+export function PendingListItem( {
+	translate,
+	productName,
+	productSlug,
+	paymentType,
+	totalCost,
+	siteId,
+	siteTitle,
+	siteDomain,
+	dateCreated,
+} ) {
 	return (
-		<Card className={ 'pending-payments__list-item' }>
-			<span className="pending-payments__list-item-wrapper">
-				<div className="pending-payments__list-item-details">
-					<div className="pending-payments__list-item-title">{ productName }</div>
-					<div className="pending-payments__list-item-purchase-type">{ paymentType }</div>
-					<div className="pending-payments__list-item-purchase-cost">{ totalCostDisplay }</div>
-					<div className="pending-payments__list-item-actions">
-						<Button primary={ true } href="/help/contact">
-							<Gridicon icon="help" />
-							<span>{ translate( 'Contact Support' ) }</span>
-						</Button>
+		<React.Fragment>
+			<PurchaseSiteHeader siteId={ siteId } name={ siteTitle } domain={ siteDomain } />
+			<Card className={ 'pending-payments__list-item is-compact' }>
+				<span className="pending-payments__list-item-wrapper">
+					<div className="pending-payments__list-item-details">
+						<div className="pending-payments__list-item-title">{ productName }</div>
+						<div className="pending-payments__list-item-product">
+							{ purchaseType( { product_slug: productSlug } ) }
+						</div>
+						<div className="pending-payments__list-item-payment">
+							{ translate(
+								'Payment of %(totalCost)s was initiated on %(dateCreated)s via %(paymentType)s.',
```

Comment:
Let's add some info about when we expect it to close. I don't mind this being hardcoded somewhere, but we can start with "up to a week" for Sofort.